sugarloaf/src/context/mod.rs

```diff
@@ -1,20 +1,22 @@
 use crate::sugarloaf::{SugarloafWindow, SugarloafWindowSize};
 use crate::SugarloafRenderer;
+use std::sync::Arc;
 
 pub struct Context<'a> {
-    pub device: wgpu::Device,
+    pub device: Arc<wgpu::Device>,
```

Comment:
Any reason to move to Arc?

Reply:
`librashader::runtime::wgpu::FilterChain::load_from_path` requires